### DISCUSSION TITLE
using puppeteer-screen-recorder in AWS lambda

### DISCUSSION THREAD

**Discussion by prasanaworld** [prasanaworld/puppeteer-screen-recorder#60] [Q&A]
Please post your queries regarding use of puppeteer-screen-recorder in aws lambda

**Comment by geetpurwar** [comment]
@prasanaworld  Thats not working because of size limitation. May I request you to create a small video regarding imp;lementation.

**Comment by mecirmartin** [comment]
Hey @prasanaworld , i am also curious if there is a way to integrate this into AWS lambda, ideally uploading these recordings into S3.
I think that would be really useful, i would gladly help with development of this feature

**Comment by ackava** [comment]
@prasanaworld I am using puppeteer recorder in AWS Lambda, but you have to use container image to do it. Its easier that way.

https://github.com/neurospeech/puppeteer-lambda-layer

You have to setup your own AWS Container Registry and setup following variables/secrets.

1. Add Variable `AWS_ACCESS_KEY_ID`
2. Add Secret `AWS_SECRET_ACCESS_KEY`
3. Setup ECR Repository with name `puppeteer-lambda`

For FFmpeg you might have to set at least 4 GB of RAM, but looking at size of chrome, ideally 6 to 8 GB would be required.
